src/transports/utils/parse_text_track.ts

```diff
@@ -45,8 +45,8 @@ export function extractTextTrackFromISOBMFF(chunkBytes : Uint8Array) : string {
 export function getISOBMFFTextTrackFormat(
   representation : Representation
 ) : "ttml" | "vtt" {
-  const codec = representation.codec == null ? "" :
-                                               representation.codec;
+  const codec = representation.codec === undefined ? "" :
```

Comment:
Why not :
`const codec = representation.codec ?? "";`

Reply:
I updated the code to actually throw an error when the codec is not known